src/status_im2/contexts/chat/composer/utils.cljs

```diff
@@ -190,15 +190,21 @@
 
 (defn init-subs
   []
-  (let [chat-input (rf/sub [:chats/current-chat-input])]
-    {:images               (seq (rf/sub [:chats/sending-image]))
-     :link-previews?       (rf/sub [:chats/link-previews?])
-     :audio                (rf/sub [:chats/sending-audio])
-     :reply                (rf/sub [:chats/reply-message])
-     :edit                 (rf/sub [:chats/edit-message])
-     :input-with-mentions  (rf/sub [:chat/input-with-mentions])
-     :input-text           (:input-text chat-input)
-     :input-content-height (:input-content-height chat-input)}))
+  (let [chat-screen-loaded? (rf/sub [:shell/chat-screen-loaded?])]
+    (merge
+     {:chat-screen-loaded? chat-screen-loaded?
+      :link-previews?      false}
```

Comment:
### Performance Note
We need to further optimize composer to improve chat loading time and performance. Maybe we can create a dummy composer as a placeholder

Reply:
probably not only composer but entire chat screen

Reply:
> We need to further optimize composer to improve chat loading time and performance.

For me, the performance is already very good on iOS and Android